modules/libcom/src/error/errSymLib.c

```diff
@@ -33,69 +35,55 @@
 static epicsUInt16 errhash(long errNum);
 
 typedef struct errnumnode {
-    ELLNODE            node;
     long               errNum;
     struct errnumnode *hashnode;
     const char        *message;
     long               pad;
 } ERRNUMNODE;
 
-static ELLLIST errnumlist = ELLLIST_INIT;
-static ERRNUMNODE **hashtable;
-static int initialized = 0;
+typedef struct {
+    ERRNUMNODE *table[NHASH * sizeof(ERRNUMNODE *)];
+    epicsMutexId tableMutexId;
+} errHashTable_t;
+
+static errHashTable_t errHashTable;
+
 extern ERRSYMTAB_ID errSymTbl;
 
+static void populateEpicsErrorSymbols()
+{
+    ERRSYMBOL *errArray = errSymTbl->symbols;
+    int i;
+
+    for (i = 0; i < errSymTbl->nsymbols; i++, errArray++) {
+        if (errSymbolAdd(errArray->errNum, errArray->name)) {
+            fprintf(stderr, "errSymBld: ERROR - errSymbolAdd() failed \n");
```

Comment:
This new `populateEpicsErrorSymbols()` is only called from `errSymBld()`, which does nothing but call `populateEpicsErrorSymbols()`.  Why introduce another function name?

Reply:
it was just for having a clearer code, the new function has a name that describes what it does... however, I'm happy to move it to `errSymBld()` if that is preferred.

Reply:
OK, I've moved it, given that the big comment at the top says what it does, it was redundant